src/Controller/OrderItemController.php

```diff
@@ -45,7 +45,8 @@ public function __construct(
         Controller\ResourceUpdateHandlerInterface $resourceUpdateHandler,
         Controller\ResourceDeleteHandlerInterface $resourceDeleteHandler,
         MessageBusInterface $messageBus
-    ) {
```

Comment:
Is this ECS change? I think PSR says 
'When the argument list is split across multiple lines, the closing parenthesis and opening brace MUST be placed together on their own line with one space between them.'
 
https://www.php-fig.org/psr/psr-12/

Reply:
it was probably lint extension in phpstorm. i will change it back.